docs/apm/traces.asciidoc

```diff
@@ -19,3 +19,18 @@ If there's a particular endpoint you're worried about, select it to view its
 
 [role="screenshot"]
 image::apm/images/apm-traces.png[Example view of the Traces overview in APM app in Kibana]
+
+[float]
+[[trace-explorer]]
+==== Trace explorer
```

Comment:
@dgieselaar there's not really any substance in the docs I've added here. Can you help me understand what trace explorer adds—why would someone want to use this feature and how can they benefit from it?